idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -0,0 +1,1799 @@
+            print('homotopy step -->{0:5.2f}'.format(i))
+            blk.homotopy_par_m = i
+            with idaeslog.solver_log(solve_log, idaeslog.DEBUG) as slc:
+                res = opt.solve(blk, tee=slc.tee)
+                if res.solver.status != SolverStatus.warning:
+                    print('')
+
+        init_log.info("Step 4 complete: {}.".format(idaeslog.condition(res)))
+        # ---------------------------------------------------------------------
+
+        print('==============================================================')
+        print('STEP 5: Adiabatic chemical absoption')
+        print("Homotopy steps:")
+        print("Isothermal (0.0) --> (1.0) Adiabatic ")
+
+        # Unfix temperature
+        blk.liquid_phase.properties[:, :].temperature.unfix()
+        blk.vapor_phase.properties[:, :].temperature.unfix()
+        # unfix heat transfer terms
+        blk.Q_v.unfix()
+        blk.Q_l.unfix()
+        blk.vapor_phase.heat.unfix()
+        blk.liquid_phase.heat.unfix()
+        # unfix enthalpy flow variable terms
+        blk.vapor_phase._enthalpy_flow[:, :, :].unfix()
+        blk.vapor_phase.enthalpy_flow_dx[:, :, :].unfix()
+        blk.liquid_phase._enthalpy_flow[:, :, :].unfix()
+        blk.liquid_phase.enthalpy_flow_dx[:, :, :].unfix()
+
+        # activate steady-state energy balance related equations
+        for c in blk.component_objects(Constraint, descend_into=True):
+            if c.local_name in [
+                "vapor_phase_heat_transfer",
+                "liquid_phase_heat_transfer",
+                "vapor_phase_heat_transfer_handle",
+                "liquid_phase_heat_transfer_handle",
+                "vap_BC_temperature",
+                "liq_BC_temperature",
+                "enthalpy_flow_linking_constraint",
+                "enthalpy_flow_dx_disc_eq",
+                    "enthalpy_balances"]:
+                c.activate()
+
+        for i in homotopy_steps_h:
+            print('homotopy step -->{0:5.2f}'.format(i))
+            blk.homotopy_par_h = i
+            with idaeslog.solver_log(solve_log, idaeslog.DEBUG) as slc:
+                res = opt.solve(blk, tee=slc.tee)
+
+        init_log.info("Step 5 complete: {}.".format(idaeslog.condition(res)))
+        # ---------------------------------------------------------------------
+
+        # scale up stripper length
+        if (blk.config.process_type == 'Stripper' or
+                blk.config.process_type == 'Regenerator'):
+            packing_height = np.linspace(0.6, specified_length, num=10)
+            print('SCALEUP Stripper height')
+            for L in packing_height:
+                blk.length_col.fix(L)
+                print('Packing height = {:6.2f}'.format(L))
+                with idaeslog.solver_log(solve_log, idaeslog.DEBUG) as slc:
+                    res = opt.solve(blk, tee=slc.tee)
+                init_log.info("Scaleup: {}.".format(idaeslog.condition(res)))
+
+        if not blk.config.dynamic:
```

Comment:
Again, printing profiles should not be part of the initialization routine. That is something for the user to do later if desired.